crates/proof-of-sql/src/base/scalar/mod.rs

```diff
@@ -68,7 +68,7 @@ pub trait Scalar:
     + std::convert::From<i32>
     + std::convert::From<i16>
     + std::convert::From<bool>
-    + TryFrom<BigInt, Error = ConversionError>
+    + TryFrom<BigInt, Error = DecimalError>
```

Comment:
This shouldn't really be a DecimalError since there are no decimals involved. It is a conversion from BigInt to Scalar.
It's ultimately used for a decimal conversion, but at this point in the code, isn't decimal related.